chukwuemekachm/Population-Management-System
# #165007750 Modifies the Prisma client schema, GraphQL Types and Project Type definitions

#### What does this PR do?
- This PR modifies the Prisma client schema, GraphQL Types and Project Type definitions
#### Description of Task to be completed?
- modify Prisma schema definition
- modify GraphQL schema definition
- modify tests
- modify fixtures
- modify Prisma Client
#### How should this be manually tested?
- Clone the repo
- Run 
```
npm install
```
- Run 
```
npm run dev
```
- Open your browser and type this into the address bar `localhost:<PORT>/graphql`
- Run 
```
npm test
```
- Observe the tests
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- [#165007750](https://www.pivotaltracker.com/story/show/165007750)
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A
